### PR TITLE
Fixing notification on-boarding task and adding alert rule signal

### PR DESCRIPTION
**Pull request by ehfeng**
Fixing notification on-boarding task to adjust for the switch to sentry-plugins for notification plugins.

Also, I added a signal for when a custom alert rule is created (ignore the default rule). This will play into on-boarding later.

### CODE REVIEW

**Review by macqueen — CHANGES_REQUESTED** [review]
This shouldn't be necessary because of https://github.com/getsentry/sentry/blob/master/src/sentry/plugins/__init__.py. What are you trying to fix?

**Review by macqueen — COMMENTED** [review]
i assume this relates to the `This will play into on-boarding later.`, but is there a reason you're not adding the onboarding task right now? i guess i'm just wondering why the signal is useful without anything listening to it

File: src/sentry/models/rule.py [hunk 1]
```diff
@@ -74,6 +75,7 @@ def save(self, *args, **kwargs):
         rv = super(Rule, self).save(*args, **kwargs)
         cache_key = 'project:{}:rules'.format(self.project_id)
         cache.delete(cache_key)
+        alert_rule_created.send(project=self.project, rule=self, sender=Rule)
```
Comment (ehfeng) [inline src/sentry/models/rule.py]: Good point. I'll move it there.

File: src/sentry/receivers/rules.py [hunk 2]
```diff
@@ -5,22 +5,26 @@
 from sentry.models import Project, Rule
 
 
+DEFAULT_RULE_LABEL = 'Send a notification for new events'
```
Comment (ehfeng) [inline src/sentry/receivers/rules.py]: I'm using this in getsentry to ignore the first signal created by the default rule.

**Comment by ehfeng** [comment]
I'll have receivers within the `getsentry` codebase to track as organizations use a feature for the first time. The pattern is very similar to the OrganizationOnboardingTask model.

**Review by macqueen — APPROVED** [review]
✨

**Pull request merged**